packages/integration-tests/test/connections.spec.ts

```diff
@@ -259,7 +259,7 @@ describe('connections', () => {
     ({ dialer, listener } = await createPeers({
       transports: [
         memory({
-          latency: 1000
+          latency: 5_000
```

Comment:
Why did the latency need to be increased here? I can't recall seeing flakiness but may be my ignorance

Reply:
It wasn't reliably triggering timeout errors.  There was flakiness in Windows in CI.